notification-center/client/angular.mdx

```diff
@@ -63,9 +71,7 @@ Component HTML template:
 The `notification-center-component` accepts the same set of props as the [web component](/notification-center/client/web-component).
 
 <Tip>
-  May need to add `"allowSyntheticDefaultImports": true` in tsconfig.json
-  and `@types/react` as dev dependency for the angular component to work
-  properly
+  You may need to add `"allowSyntheticDefaultImports": true` in tsconfig.json for the web component's React imports to work correctly.
```

Comment:
No need to install `@types/react` anymore per https://github.com/novuhq/novu/pull/4518.